packages/error-reporting/package.json

```diff
@@ -0,0 +1,56 @@
+{
+  "name": "@google-cloud/error-reporting",
+  "description": "Stackdriver Error Reporting Client Library for Node.js",
+  "main": "index.js",
+  "repository": "GoogleCloudPlatform/google-cloud-node",
+  "scripts": {
+    "test": "nyc --exclude=\"fuzzer.js\" $(npm bin)/mocha ./test/unit/*.js",
```

Comment:
> It may be easier than to simply run the test sets separately as this would require some fairly large restructuring, at least at first. I can make the other changes cited but given our timeline it might be better to keep these separated for the time being

We have a single test command right now, so that is really tough to do.